server/belga/io/feed_parsers/belga_afp_newsml_1_2.py

```diff
@@ -10,13 +10,58 @@
 
 from superdesk.io.registry import register_feed_parser
 from .base_belga_newsml_1_2 import BaseBelgaNewsMLOneFeedParser
+from superdesk import get_resource_service
+from lxml import etree
 
 
 class BelgaAFPNewsMLOneFeedParser(BaseBelgaNewsMLOneFeedParser):
     """Feed Parser for Belga specific AFP NewsML."""
 
     NAME = 'belga_afp_newsml12'
     label = 'Belga specific AFP News ML 1.2 Parser'
+    MAPPING_KEYWORDS = {
+        'ECONOMY': ['BOURSE', 'ECONOMIE', 'CONOMIE', 'MARCHES', 'FINANCE', 'BANQUE'],
+        'SPORTS': ['HIPPISME', 'SPORT', 'SPORTS']
+    }
+    MAPPING_CATEGORY = {
+        'SPO': 'SPORT',
+        'POL': 'POLITICS',
+    }
+
+    def parse(self, xml, provider=None):
+        def add_unique_item_to_list(data, list):
```

Comment:
We think it better for list to remain order, because the UI load only first news_products item to editor and all of other tests case use assertEqual for `item['subject']`

Reply:
I think we don't care much about ordering, `news_products` defined as multiselection, you can see it in a `belga_text` content profile. IMO this type of data should not care about ordering. 
![Screenshot 2019-10-09 at 11 36 07](https://user-images.githubusercontent.com/4199922/66470243-7ff9b080-ea89-11e9-8f2b-5da92c3cc516.png)

Reply:
Agree, my bad forgot about `set` part, can't use `set` here because this is list of dict